AEXS/aex-141.md

```diff
@@ -13,21 +13,23 @@ Created: 2021-09-11
 
 ## Abstract
 
-A standard implementation of non-fungible tokens for the Aeternity ecosystem. The design goal of the primary interface is to be as compatible with ERC-721 as possible, so that anyone who can work with ERC-721 can work with this interface. However, where Sophia offers a better way, performance and efficiency should prevail over compatibility.
+A standard implementation of non-fungible tokens for the æternity ecosystem. The design goal of the primary interface is to be as compatible with ERC-721 as possible, so that anyone who can work with ERC-721 can work with this interface. However, where Sophia offers a better way, performance and efficiency should prevail over compatibility.
 
 There is no support for unsafe transactions. Therefore all transactions are ought to be safe.
 
+For minting it is expected that different options will evolve over time. The way minting is done has to be defined via extensions.
+
 ## Motivation
 
-The following standard describes standard interfaces for non-fungible tokens. The proposal contains a primary interface and secondary interfaces for optional functionality that not everyone might need. 
+The following standard describes standard interfaces for non-fungible tokens. The proposal contains a primary interface and secondary interfaces (extensions) for optional functionality that not everyone might need.
 
-# Basic NFT
+# AEX141 NFT
 
-## Interface
+## IAEX141
 
 ```sophia
-contract interface NFT =
-    datatype metadata_type = URL | IPFS | OBJECT_ID | MAP
```

Comment:
is ipfs now supposed to be `OBJECT_ID`?

Reply:
IPFS is now supposed to be `URL` (e.g. `ipfs://`) like we do in metadata, too.